internal/store/ring.go

```diff
@@ -16,17 +17,40 @@ var ringSelect sq.SelectBuilder
 
 func init() {
 	ringSelect = sq.
-		Select("Ring.ID", "Name", "Priority", "SoakTime", "Image", "Version", "Provisioner", "State", "CreateAt", "DeleteAt", "ReleaseAt", "APISecurityLock", "LockAcquiredBy", "LockAcquiredAt").
+		Select("Ring.ID", "Name", "Priority", "SoakTime", "Image", "Version", "Provisioner", "ChangeRequestMetadataRaw", "State", "CreateAt", "DeleteAt", "ReleaseAt", "APISecurityLock", "LockAcquiredBy", "LockAcquiredAt").
 		From("Ring")
 }
 
 type rawRing struct {
 	*model.Ring
+	*RawRingMetadata
+}
+
+// RawRingMetadata is the raw byte metadata for a ring.
+type RawRingMetadata struct {
+	ChangeRequestMetadataRaw []byte
```

Comment:
why do you need this to be a byte? The DB is sqlite or postgres? Because postgres support json column type as a field. Just to be more readable in Database if it's needed.

Reply:
DB that will be used in Production will be Postgres but for local development we use sqlite. Similar to what we do with provisioner.